packages/core/src/commands/snapshot-truncate.ts

```diff
@@ -35,8 +36,8 @@ export class Command extends Commands.Command {
         this.definition
             .setFlag("token", "The name of the token.", Joi.string().default("ark"))
             .setFlag("network", "The name of the network.", Joi.string().valid(...Object.keys(Networks)))
-            .setFlag("skipCompression", "Skip gzip compression.", Joi.boolean())
-            .setFlag("trace", "Dumps generated queries and settings to console.", Joi.boolean());
+            .setFlag("skipCompression", "Skip gzip compression.", Joi.boolean());
+        // .setFlag("trace", "Dumps generated queries and settings to console.", Joi.boolean());
```

Comment:
Is this now unused or will it be implemented?

Reply:
Unused and removed.